src/meili/map_matching.cc

```diff
@@ -214,5 +232,156 @@ inline double
 MapMatching::CostSofar(double prev_costsofar, float transition_cost, float emission_cost) const
 { return prev_costsofar + transition_cost + emission_cost; }
 
+
+TransitionCostingModel::TransitionCostingModel(
```

Comment:
An implementation of `ITransitionCostingModel` that works with real states (candidates)